dice-game-ui-angular/src/app/start-game/start-game.component.ts

```diff
@@ -1,36 +1,44 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
-
 import { SimpleGameEngine } from 'engine';
 import { GameStorageService } from '../services/game-storage-service';
 import { StatisticsService } from '../services/statistics-service';
+import { UserService } from '../services/user-service';
 
 @Component({
   selector: 'app-start-game',
   templateUrl: './start-game.component.html',
   styleUrls: ['./start-game.component.scss'],
 })
 export class StartGameComponent implements OnInit {
-  public size: number;
-  public sizeOptions: number[] = [];
+  size: number;
+  sizeOptions: number[] = [];
+
+  private readonly localPlayer = 'local player';
```

Comment:
Local player is not needed here anymore.

Reply:
Removed